docs/content/en/docs/concepts/architecture/components/scheduler/_index.md

```diff
@@ -6,11 +6,50 @@ weight: 80
 cascade:
 ---
 
-**Keptn's Lifecycle Scheduler** replaces the
-[Kubernetes scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/)
-to allow users to schedule events and tasks to occur
-at specific times during the application lifecycle.
-The Lifecycle Scheduler can trigger events such as
-deployment, testing, and remediation at specific times or intervals.
-The Keptn Scheduler guarantees that Pods are initiated only after
-the Pre-Deployment checks are completed.
+The **Keptn Scheduler**, an integral component of the Keptn Lifecycle Toolkit which orchestrates
+the deployment process.
+The **Keptn Scheduler** works by registering itself as a Permit plugin with Kubernetes that
+is responsible for ensuring that Pods are scheduled to a node until and unless the
+pre-deployment checks have finished successfully.
+This is important because it helps to prevent Pods from being scheduled to nodes that are
+not yet ready for them, which can lead to errors and downtime
+
+## How does the Keptn Scheduler works
+
+The Keptn Scheduler first checks the annotations on the Pod to see if it is annotated with
+[Keptn specific annotations](https://main.lifecycle.keptn.sh/docs/implementing/integrate/#basic-annotations).
+If the Pod is annotated with Keptn specific annotations, the Keptn Scheduler retrieves
+the WorkloadInstance CRD that is associated with the Pod.
+The **WorkloadInstance CRD** contains information about the `pre-deployment` checks that
+need to be performed before the Pod can be scheduled.
+
+The Keptn Scheduler then checks the status of the WorkloadInstance CRD to see if the
+`pre-deployment` checks have finished successfully.
+If the pre-deployment checks have finished successfully, the **Keptn Scheduler** allows
+the Pod to be scheduled to a node.
```

Comment:
I would also add here that since the scheduler is a plugin to the default k8s scheduler, all other checks that are usually performed by it apply here, for instance, if no node has enough capacity for the deployment this will not be scheduled whether our pre-deployment phase succeded or not.